app/lib/screens/faq/Item.dart

```diff
@@ -15,16 +16,357 @@ class FaqItemScreen extends StatefulWidget {
+                            width: MediaQuery.of(context).size.width,
+                            height: 40,
+                            decoration: BoxDecoration(
+                              color: Colors.black,
+                              borderRadius: BorderRadius.circular(15),
+                            ),
+                            child: Stack(
+                              alignment: Alignment.centerRight,
+                              children: <Widget>[
+                                Padding(
+                                  padding: const EdgeInsets.only(left: 8.0),
+                                  child: TextField(
+                                    style: const TextStyle(color: Colors.white),
+                                    cursorColor: Colors.grey,
+                                    controller: _controller,
+                                    decoration: const InputDecoration(
+                                      hintText: 'Add a comment',
+                                      hintStyle: TextStyle(
+                                        color: Colors.grey,
+                                      ),
+                                      border: InputBorder.none,
+                                    ),
+                                  ),
+                                ),
+                                IconButton(
+                                  icon: const Icon(
+                                    Icons.emoji_emotions_outlined,
+                                    color: Colors.grey,
+                                  ),
+                                  onPressed: () {
+                                    setState(() {
+                                      emojiShowing = !emojiShowing;
+                                    });
+                                  },
+                                ),
+                              ],
+                            ),
+                          ),
+                        ),
+                        Flexible(
+                          flex: 1,
+                          child: SizedBox(
+                            width: 30,
+                            child: IconButton(
+                              onPressed: () {
+                                const snackBar = SnackBar(
+                                  content: Text('Send icon tapped'),
+                                );
+                                ScaffoldMessenger.of(context)
+                                    .showSnackBar(snackBar);
+                              },
+                              icon: const Icon(
+                                Icons.send,
+                                color: Colors.pink,
+                              ),
+                            ),
+                          ),
+                        )
+                      ],
+                    ),
+                    Row(
+                      children: const [
```

Comment:
these don't really do anything, do they? Let's comment them out and file a ticket about that then..